controller/space_iterations_test.go

```diff
@@ -284,15 +281,15 @@ func (rest *TestSpaceIterationREST) TestWICountsWithIterationListBySpace() {
 	iteration1 := iteration.Iteration{
 		Name:       "Sprint 1",
 		SpaceID:    spaceInstance.ID,
-		UserActive: &userActive,
+		UserActive: false,
 	}
 	iterationRepo.Create(rest.ctx, &iteration1)
 	assert.NotEqual(rest.T(), uuid.UUID{}, iteration1.ID)
 
 	iteration2 := iteration.Iteration{
 		Name:       "Sprint 2",
 		SpaceID:    spaceInstance.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c